src/Controller/Backend/ContentEditController.php

```diff
@@ -512,6 +512,10 @@ private function getEditLocale(Request $request, Content $content): string
             $locale = $content->getDefaultLocale();
         }
 
+        if (! $locale) {
+            $locale = $this->defaultLocale;
```

Comment:
If we rebase this on master, line 508 says: 

```php
  $locale = $this->getFromRequest('edit_locale', '');
```

Couldn't we just do this? 

```php
  $locale = $this->getFromRequest('edit_locale', $this->defaultLocale);
```

Or would that conflict with the `$locales->contains($locale)` below it?

Reply:
It will conflict with the `$locales->contains($locale)`. This fix is meant for cases when the content locales are not defined. In that case, `$locales` will always be empty, but `$content->getDefaultLocale()` will return null. The idea is to avoid returning null when no locales are defined.

Reply:
Ok, in that case: let’s roll with this! :-)